### PR TITLE
Add XLA specific XPlane profiling.

### PR DESCRIPTION
**Pull request by golechwierowicz**
Following internal discussion with @vanbasten23 I'm adding an option to use the native XLA profiler. The exported xplane file can be opened in the internal tooling, and it shows what is happening on the host side. That's in contrast to the native pytorch profile we open via perfetto.

### CODE REVIEW

File: benchmarks/experiment_runner.py [hunk 1]
```diff
@@ -906,6 +932,11 @@ def main():
     logger.warning('Enabling fast F32 multiplication for PyTorch')
     torch.set_float32_matmul_precision('high')
 
+  if args.profile_xla:
+    logger.info(
+        'Enabling XLA XPlane profiling. Do not benchmark with this option set.')
+    server = xp.start_server(9012)
```
Comment (vanbasten23) [inline benchmarks/experiment_runner.py:938]: consider adding a FLAGS.profiler_xla_port default to 9012

**Review by vanbasten23 — APPROVED** [review]
LGTM. Thanks!

File: benchmarks/experiment_runner.py (hunk 1 above)
Comment (golechwierowicz) [reply benchmarks/experiment_runner.py:938]: Do you mean importing some specific constant, or parametrising server port?

Comment (vanbasten23) [reply benchmarks/experiment_runner.py:938]: I meant parametrising server port, such as https://github.com/pytorch/xla/blob/dc08cb6893e84f29fc31ed4e44e8998ed680b2a1/test/spmd/test_train_spmd_imagenet.py#L380

**Pull request merged**